uat/steps/component.py

```diff
@@ -0,0 +1,69 @@
+import os
+import t_utils
+
+from behave import step
+from pathlib import Path
+from constants import (
+    GG_CONFIG_JSON, GG_RECIPE_YAML, GG_BUILD_DIR, GG_BUILD_ZIP_DIR,
+    DEFAULT_AWS_REGION, DEFAULT_S3_BUCKET_PREFIX, DEFAULT_ARTIFACT_AUTHOR
+)
+
+
+@step('we verify component files')
```

Comment:
We verify gdk project files just to avoid confusion as we usually consider component files as greengras-build artifacts and recipes.

Reply:
done

Reply:
done